### PR TITLE
get go-getter v2.0.2 to avoid untaring OVA files

### PR DESCRIPTION
**Pull request by azr**
related: 

* https://github.com/hashicorp/packer/issues/11631
* https://github.com/hashicorp/go-getter/pull/354

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -38,7 +38,7 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
 	github.com/hashicorp/go-getter/gcs/v2 v2.0.0-20200604122502-a6995fa1edad // indirect
 	github.com/hashicorp/go-getter/s3/v2 v2.0.0-20200604122502-a6995fa1edad // indirect
-	github.com/hashicorp/go-getter/v2 v2.0.0 // indirect
+	github.com/hashicorp/go-getter/v2 v2.0.2 // indirect
```
Comment (sylviamoss) [inline go.mod:41]: why is it indirect? 🤔

**Review by sylviamoss — APPROVED** [review]
👍🏼

File: go.mod (hunk 1 above)
Comment (azr) [reply go.mod:41]: 🤔 I wonder, maybe because Packer core requires it ?

Comment (azr) [reply go.mod:41]: Yup and actually this plugin does not require the go-getter

**Pull request merged**